.env.example

```
@@ -25,7 +25,12 @@ RESEND_API_KEY=
 RECAPTCHA_SECRET_KEY=
 X_BEARER_TOKEN=
 
-# Deploy-only secrets (GitHub Actions deploy.yml)
-GCP_SSH_KEY=
-GCP_HOST=
-GCP_USER=deploy
+# Runtime secrets (Cloudflare Workers Secrets — set via `wrangler secret put <NAME>`)
+RESEND_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The RESEND_API_KEY key is duplicated

(DuplicatedKey)